examples/contour/02-role-contour.yaml

```diff
@@ -48,61 +48,61 @@ rules:
   verbs:
   - list
 - apiGroups:
-  - networking.k8s.io
-  resources:
-  - gatewayclasses
-  - gateways
-  - httproutes
-  - tcproutes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - networking.k8s.io
+  - contour_api_v1.io
```

Comment:
looks like this file needs to be reverted

Reply:
👍